src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -235,17 +242,24 @@ private static string GetSymbolsPath(string packagePath, bool isSnupkg)
                 Path.GetFileName(packageToPush),
                 sourceName));
 
+            bool showPushCommandPackagePushed = true;
```

Comment:
better name would have been something like "packagePushStatus". 
Variable should be named based on what they represent, not how they are used.